lib/ansible/module_utils/facts.py

```diff
@@ -1943,6 +1944,35 @@ def get_dmi_facts(self):
         rc, out, err = self.module.run_command("/usr/sbin/lsattr -El sys0 -a fwversion")
         data = out.split()
         self.facts['firmware_version'] = data[1].strip('IBM,')
+        rc, out, err = self.module.run_command("/usr/sbin/lsconf")
```

Comment:
use `self.module.get_bin_path('lsconf')` and make the command run conditional on that, we are moving to this in all modules to avoid hardcoding paths of binaries that might be moved. 

refactoring facts.py to use this universally is on my list as it has many instances of not following this rule.